build/notarize.js

```diff
@@ -7,6 +7,10 @@ exports.default = async function notarizing(context) {
     return;
   }
 
+  if(!process.env.APPLEID || !process.env.APPLEIDPASS){
+    return
+  }
```

Comment:
Apple developer account is not a mandatory. After committing this, it would be mandatory.